services/petstore/cmd/controller/main.go

```diff
@@ -14,8 +14,6 @@
 package main
 
 import (
-	"os"
```

Comment:
Looks like this was accidentally removed, though?

Reply:
yea fixed